external/a2a/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@microsoft/teams.a2a",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "homepage": "https://github.com/microsoft/teams.ts",
+  "bugs": "https://github.com/microsoft/teams.ts/issues",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/teams.ts.git",
+    "directory": "packages/a2a"
+  },
+  "keywords": [
+    "microsoft",
+    "teams",
+    "msteams"
+  ],
+  "scripts": {
+    "clean": "npx rimraf ./dist",
+    "build": "npx tsc",
+    "test": "npx jest",
+    "lint": "npx eslint",
+    "lint:fix": "npx eslint --fix"
+  },
+  "peerDependencies": {
+    "@microsoft/teams.common": "*",
```

Comment:
these will need to be set to the latest package version since dependencies need to be explicit when using prerelease versions.

Reply:
ah TIL. thanks!